core/src/main/java/io/questdb/cairo/sql/async/PageFrameReduceJob.java

```diff
@@ -193,7 +194,11 @@ private static boolean consumeQueue(
                         // If this is not work stealing, or the stealing frame sequence is not working on its own task,
                         // we need to initialize the circuit breaker wrapper with the task's circuit breaker.
                         if (workerId != -1 || frameSequence != stealingFrameSequence) {
-                            circuitBreaker.init(frameSequence.getCircuitBreaker());
+                            circuitBreaker.backup();
```

Comment:
can we document the backup behaviour please?

Reply:
Done in 11abcb858ccbf6a31f3a80e56ec16f9275063d73